From c8146f140590cee8d3fbf9c571bce269836ee123 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Sun, 7 Jan 2018 17:24:42 +0200
Subject: [PATCH 1309/1345] fix: dts: set 8040-mcbin phy-mode of mac3 in cp1
 to 1000base-x

commit  7f00164333497b3cef91439ef060b17956e1025a from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

CP110-1 mac3 phy-mode fixed to correct "1000base-x"

Change-Id: I6b3c0488b2591cc4f7dbf563809302ef783b7d06
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/48611
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/49809
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 94cb12a..831297a 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -165,7 +165,7 @@
 					phy = <&smi_phy0>;
 				};
 				emac3_1: mac3 {
-					phy-mode = "sgmii"; /* lane-5 */
+					phy-mode = "1000base-x"; /* lane-5 */
 					phy-speed = <2500>;
 					phys = <&comphy1 5 COMPHY_HS_SGMII2>;
 					phy-names = "comphy";
-- 
1.7.9.5

